Versions/Common/logic.zone.lua

```diff
@@ -99,11 +99,13 @@ end
 function BeStride:IsSpecialZone()
 	local mapID = C_Map.GetBestMapForUnit("player")
 	local micro = BeStride:GetMapUntil(mapID,5)
-	local dungeon = BeStride:GetMapUntil(mapID,4)
-	local zone = BeStride:GetMapUntil(mapID,3)
 	local continent = BeStride:GetMapUntil(mapID,2)
+
+	if continent == nil or micro == nil then
+		return false
+	end
 	
-	if continent ~= nil and continent.name == LibStub("AceLocale-3.0"):GetLocale("BeStride")["Continent.Draenor"] and micro.name == LibStub("AceLocale-3.0"):GetLocale("BeStride")["Zone.Nagrand"] and self:DBGet("settings.mount.telaari") == true then
+	if continent.name == LibStub("AceLocale-3.0"):GetLocale("BeStride")["Continent.Draenor"] and micro.name == LibStub("AceLocale-3.0"):GetLocale("BeStride")["Zone.Nagrand"] and self:DBGet("settings.mount.telaari") == true then
```

Comment:
If you haven't unlocked your Nagrand zone ability, then `micro` comes back `nil` and indexing into it here breaks.

Reply:
Was having the micro issue on my warlock which doesn't have flying yet. Monkey patched the above code in and just added an additional `micro ~= nil` check into the condition and works fine for now